app.js

```diff
@@ -0,0 +1,12 @@
+const express = require("express");
+const { getTopics } = require("./controllers/topics.controllers");
+const app = express();
+
+app.use(express.json());
+
+app.get("/api/topics", getTopics);
+
+app.use((err, req, res, next) => {
```

Comment:
you will want to console.log the error here for when an error eventually gets to the error handling middleware